GPU/GPUTracking/Global/GPUChainTrackingClusterizer.cxx

```diff
@@ -641,6 +641,21 @@ int32_t GPUChainTracking::RunTPCClusterizer(bool synchronizeOutput)
   const GPUSettingsProcessingNNclusterizer& nn_settings = GetProcessingSettings().nn;
   GPUTPCNNClusterizerHost nnApplications[GetProcessingSettings().nTPCClustererLanes];
 
+  // Maximum of 4 lanes supported
+  HighResTimer* nnTimers[12] = {
```

Comment:
Could you obtain the timers only if
```
if (GetProcessingSettings().nn.applyNNclusterizer && GetProcessingSettings().debugLevel >= 1)
```
? You can still declare them unconditionally, but fill the array with valid timers only in an if condition.